### PR TITLE
159239068 Add Development modules to gitignore Files

### PR DESCRIPTION
**Pull request by koiic**
#### relevant pivotal tracker story

[#159239068](<https://www.pivotaltracker.com/n/projects/2183409/stories/159239068>)

### CODE REVIEW

**Comment by fidelisojeah** [comment]
For the Relevant Pivotal Tracker Story, you only need to put the link to the story 
e.g
`[#159239068](<https://www.pivotaltracker.com/story/show/159239068>)`

This would appear as:
[#159239068](<https://www.pivotaltracker.com/story/show/159239068 >)

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,6 @@ script:
 
 after_script:
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
-  - NODE_ENV=test istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
 
 
 after_success:  npm run coveralls
```
Comment (fidelisojeah) [inline .travis.yml:19]: Try removing this line
Your travis may pass afterwards

**Review by fidelisojeah — CHANGES_REQUESTED** [review]
For the Relevant Pivotal Tracker Story, you only need to put the link to the story 
e.g
`[#159239068](<https://www.pivotaltracker.com/story/show/159239068>)`

This would appear as:
[#159239068](<https://www.pivotaltracker.com/story/show/159239068>)

File: .travis.yml (hunk 1 above)
Comment (koiic) [reply .travis.yml:19]: Okay

**Comment by fidelisojeah** [comment]
You can merge this.

**Pull request merged**